include/nsysnet/netconfig.h

```diff
@@ -198,12 +198,12 @@ WUT_CHECK_SIZE(NetConfOpt, 0x2c1);
 
 struct WUT_PACKED NetConfInterface {
     uint16_t if_index;
-    uint16_t if_sate;
+    uint16_t if_state;
```

Comment:
FWIW the network configuration name this modifies is called `if_sate`, probably a typo on the OS side. Unsure if we should change this or not.

Reply:
How about this then:
```cpp
union {
    uint16_t if_sate;
    uint16_t if_state;
};
```
That way both are available.

Reply:
I think using an union here is unnecessary. We should either fix the typo or keep it consistent with the OS names.